api/spec/requests/spree/api/v2/storefront/taxons_spec.rb

```diff
@@ -65,6 +65,18 @@
       end
     end
 
+    context 'by parent permalink' do
+      before { get "/api/v2/storefront/taxons?filter[parent_permalink]=#{taxonomy.root.permalink}" }
+
+      it_behaves_like 'returns 200 HTTP status'
+
+      it 'returns children taxons by parent' do
+        expect(json_response['data'].size).to eq(2)
+        expect(json_response['data'][0]).to have_relationship(:parent).with_data('id' => taxonomy.root.id.to_s, 'type' => 'taxon')
```

Comment:
@pmilewski this test will is false-positive, we need to test it does not return other taxons from this store with different parent, current state: we only have 2 taxons with the same parent, we need a 3rd one with a different parent to see if this one won't be returned